com.governikus.ausweisapp2.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>com.governikus.ausweisapp2</id>
```

Comment:
I think my main problem with this ID is that it points to domain that does not exist. Unless it's used internally for something with d-bus, I'd rather see it renamed to `de.governikus.ausweisapp2` or even `de.bund.ausweisapp.ausweisapp2` if it's considered official government app.

Reply:
Changed to `de.bund.ausweisapp.ausweisapp2`. I had to adapt in in multiple locations so everything is lined up, again. As far as I can tell by grepping through the sources, `com.governikus.ausweisapp2` only plays a role in the Android and iOS versions.